documentation/modules/ROOT/pages/021-initial-dir-and-files.adoc

```diff
@@ -158,12 +175,6 @@ cat > group_vars/pro/configure_connection_controller_credentials.yml <<EOF
 include::../examples/group_vars/pro/configure_connection_controller_credentials.yml[]
 EOF
 ----
-======
-
-[IMPORTANT]
-======
-The previous files should be encrypted as they contain sensitive information and is not recommended to store it in plain text. In our case we will use `ansible-vault`
-======
```

Comment:
I'd prefer to maintain these lines...

Reply:
they were moved to the beginning of the section for clarity on the process